Resources/Prototypes/Entities/Markers/Spawners/mobs.yml

```diff
@@ -539,7 +539,7 @@
       - MobSlimesPet
 
 - type: entity
-  name: Pun Pun Spawner
+  name: Bartender Animal Spawner
   id: SpawnMobMonkeyPunpun
```

Comment:
I would suggest you rename this as well and add a migration to change the old id to the new one

https://github.com/space-wizards/space-station-14/blob/master/Resources/migration.yml